blog/migrations/__init__.py

```diff
@@ -0,0 +1 @@
+"""Migration scripts for the stats app."""
```

Comment:
Is the init file needed here? Where are the actual migrations? They must be checked into the version control.

Reply:
Marking as resolved.